Cargo.toml

```diff
@@ -2534,6 +2534,17 @@ description = "Systems run in parallel, but their order isn't always determinist
 category = "ECS (Entity Component System)"
 wasm = false
 
+[[example]]
+name = "observers"
+path = "examples/ecs/observers.rs"
+doc-scrape-examples = true
+
+[package.metadata.example.observers]
+name = "Observers"
+description = "Demonstrates observers that react to events (both built-in life-cycle events and custom events)"
+category = "ECS (Entity Component System)"
+wasm = false
```

Comment:
any reason this is marked as now wasm compatible? don't observer work there?

Reply:
They do work there. Presumably this was because some of the ECS examples are opted out (likely because they aren't visually interesting?). I've set this true because its a cool one!